i18n/zh-cn.toml

```diff
@@ -65,3 +65,9 @@ other = "最后修改"
 other = "Redirecting..."
 [redirect_manually]
 other = "Click here if you are not redirected"
+
+# Error pages
+[error_404_title]
```

Comment:
Is there a reason we are filling out the `zh-cn` translation files as well as plain `en`?

Reply:
As I understand from Joost there's a plan to have zh-cn in the future so he started this file already so I thought it would be best to have all the overall translation keys here as well so we only need to translate in the future. If this is not the case we can also remove this file.

Reply:
The config was added because the original content for alpha had chinese as well (ie. content taken from current site).
Later on the content for zh-ch got removed from alpha.

So maybe we should just remove this config file as well?

Reply:
Yes, OK to delete this config file from my end.  

@myitcv any objections to this?

Reply:
I suggest we start from a clean state for now, i.e. just have `en` for now.

Reply:
I deleted the zh-cn.toml file. The other zh-cn toml file for the menu I will delete when I will work on the footer pr feedback in #297